draft-ietf-ppm-dap.md

```diff
@@ -114,6 +114,27 @@ seen in the clear by any server.
 
 (\*) Indicates a change that breaks wire compatibility with the previous draft.
 
+05:
+
+- Bump draft-irtf-cfrg-vdaf-05 to 06 {{!VDAF}}. (\*)
+
+- Specialize the protocol for two-party VDAFs (i.e., one Leader and One
+  Helper). Accordingly, update the aggregation sub-protocol to use the new
+  "ping-pong" interface for two-party VDAFs introduced in
+  draft-irtf-cfrg-vdaf-06. (\*)
+
+- Require the following actions to be idempotent: aggregation job creation,
+  collection job creation, and requesting the Helper's aggregate share.
```

Comment:
I don't see that collection job creation is required to be idempotent. (agg job creation & agg shares are explicitly requried to be so)

Reply:
Taking another look, I think agg job creation is also not required to be idempotent; the only related text I can find is:

> Changing an aggregation job's parameters is illegal, so further requests to PUT /tasks/{tasks}/aggregation_jobs/{aggregation-job-id} for the same aggregation-job-id but with a different AggregationJobInitReq in the body MUST fail with an HTTP client error status code.

... and this text allows, but does not require, idempotency.

Reply:
Why doesn't this require idempotency?

Reply:
It says that additional `PUT` requests to the same aggregation job ID but otherwise different parameters `MUST` fail (which is sensible & good). For idempotency to be required, we would need a different condition: that repeated identical requests (same aggregation job ID, same additional parameters) `MUST` succeed.

Reply:
Fixed, PTAL and make sure it's correct. (Also feel free to suggest a clearer description.)